### PR TITLE
Add recipe for ImageMagick version 6.9.3

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: packages/package_imagemagick_6_9_3/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="imagemagick" version="6.9.3">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url" sha256sum="5617e14d624be393209d77674b935165bddc797e65661774f81785f6c05ca0e9">
+                    https://depot.galaxyproject.org/software/ImageMagick/ImageMagick_6.9.3_src_all.tar.gz
+                </action>
+                <action type="autoconf">--prefix=$INSTALL_DIR</action>
```
Comment (bgruening) [inline packages/package_imagemagick_6_9_3/tool_dependencies.xml]: autoconf uses this prefix by default

Comment (gregvonkuster) [inline packages/package_imagemagick_6_9_3/tool_dependencies.xml]: Ah, yes!  Sorry.  Just a sec...

**Comment by gregvonkuster** [comment]
Fixed here https://github.com/gregvonkuster/tools-iuc/commit/83ef1bd3b0df676ea757475c1b371c4153cc626e

**Comment by bgruening** [comment]
have a look at the TSs

**Comment by gregvonkuster** [comment]
awesome - thanks!! ;)

**Pull request merged**